### DISCUSSION TITLE
What are procedures on upgrading javaparser-core.jar?

### DISCUSSION THREAD

**Discussion by IdeaHunter** [paulirwin/JavaToCSharp#130] [General]
I did the following:
0) Bumped IKVM to newer version and verified its working and have no performance degradations
1) Compiled new version from javaparser github
2) Got javaparser-core-3.26.3.jar
3) Replaced file and reference to it in JavaToCSharp project file
4) Cleaned and rebuild solution

Although its working, for sore reason its magnitude slower that javaparser-core-3.25.4.jar

Did i missed some steps?

**Comment by paulirwin** [comment]
Hi, sorry I missed this discussion when you originally posted it. Better late than never...

I have upgraded javaparser via now using IKVM's MavenReference, although I don't expect this to be a permanent solution. I'd like to bring Java parsing in-app via ANTLR at some point, but that's a large undertaking. For now, I've got it upgraded to the latest version for the 4.0 release that I'm finishing up now.

**Comment by IdeaHunter** [comment]
Thank you for your work! Closing issue
